src/fheroes2/audio/audio_manager.cpp

```diff
@@ -552,14 +587,16 @@ namespace
             return;
         }
 
+        // TODO: use another containers for sound effects to support more efficient sort and find operations based on the code below.
```

Comment:
Either the suggested change or `use other containers`.
```suggestion
        // TODO: use another container for sound effects to support more efficient sort and find operations based on the code below.
```